plugin/src/main/java/org/jetbrains/research/refactoringminer/OutputRefactoringHandler.java

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.research.refactoringminer;
+
+import org.jetbrains.annotations.NotNull;
+import org.refactoringminer.api.Refactoring;
+import org.refactoringminer.api.RefactoringHandler;
+import org.refactoringminer.rm1.GitHistoryRefactoringMinerImpl;
+import java.io.IOException;
+import java.util.List;
+
+public class OutputRefactoringHandler extends RefactoringHandler {
```

Comment:
what is the purpose of this class? i'm not sure about the choice of the name

Reply:
To handle finished commit or exceptions and print them